docs/cloud/insights/events-feed.md

```diff
@@ -0,0 +1,59 @@
+<!--
+title: "Events feed"
+sidebar_label: "Events feed"
+custom_edit_url: "https://github.com/netdata/netdata/blob/master/docs/cloud/insights/events-feed.md"
+sidebar_position: "2800"
+learn_status: "Published"
+learn_topic_type: "Concepts"
+learn_rel_path: "Concepts"
+learn_docs_purpose: "Present the Netdata Events feed."
+-->
+
+Netdata Cloud provides the Events feed which is a powerful feature that tracks events that happen on your infrastructure, or in your Space. The feed lets you investigate events that occurred in the past, which is obviously invaluable for troubleshooting. Common use cases are ones like when a node goes offline, and you want to understand what events happened before that. A detailed event history can also assist in attributing sudden pattern changes in a time series to specific changes in your environment.
+
+#### What are the available events?
```

Comment:
Check the level of these headings. I think they need to start at `##`

Reply:
@cakrit was following these contribution guidelines https://learn.netdata.cloud/docs/nightly/contribute/to-documentation#encapsulate-topics-into-markdown-files

Reply:
but will change since these look like this now
![image](https://user-images.githubusercontent.com/82235632/217616670-a7475d34-e25f-4d28-9d4e-5752609349d2.png)

Reply:
Ok, I will revisit the guidelines. I think I'll do away with the whole concepts, references, tasks purism.